Gems/AWSCore/Code/Source/Editor/Attribution/AWSCoreAttributionMetric.h

```diff
@@ -28,7 +28,10 @@ namespace AWSCore
         ~AttributionMetric() = default;
 
         void SetO3DEVersion(const AZStd::string& version);
+
+        AZStd::string Platform() const { return m_platform;}
```

Comment:
NP: GetPlatform() would IMO be clearer considering SetPlatform exists.

Reply:
Returning by value as return value optimization hides most of cost and code not on critical path. Returning const ref, does not guarantee constness (caller can cast it away) and because object can be mutated before the caller had any chance to copy it. However I see we often return strings via const refs so will move to this pattern.

Will update to GetPlatform.

Reply:
It looks like the update to this file wasn't committed? The unit test now shows GetPlatform while this is still Platform.